framework/sessions/__init__.py

```diff
@@ -85,12 +85,17 @@ def prepare_private_key():
 #     return user_session
 
 def get_session():
-    session_key = request.cookies.get(settings.COOKIE_NAME)
-    if session_store.exists(session_key=session_key):
-        user_session = session_store.get(session_key)
-    else:
-        user_session = session_store.create()
-    return user_session
+    secret = settings.SECRET_KEY
+    cookie = request.cookies.get(settings.COOKIE_NAME)
+    if not cookie:
+        s = SessionStore()
+        s.create()
+        return s
+    try:
+        session_key = ensure_str(itsdangerous.Signer(secret).unsign(cookie))
+        return SessionStore(session_key=session_key)
+    except itsdangerous.BadSignature:
+        return None
```

Comment:
Interesting, so SessionStore() actually inits and returns a session object ...

Reply:
Yes. A `SessionStore` object.